src/emu/drivers/src/graphics/backend/ogl/graphics_ogl.cpp

```diff
@@ -48,7 +48,7 @@ namespace eka2l1::drivers {
         glGetIntegerv(GL_MAJOR_VERSION, &major_gl);
 
         if (!is_gles && (major_gl < 4)) {
-            std::list<std::string> GL_REQUIRED_EXTENSIONS = {
+            std::vector<std::string> GL_REQUIRED_EXTENSIONS = {
```

Comment:
Better to be an array like you did before

Reply:
@pent0 fix it https://github.com/EKA2L1/EKA2L1/commit/e69502495791ac4f7c99fa21d58ad16deb6e435e